### PR TITLE
Added a page with jobs

### PR DESCRIPTION
**Pull request by bsekachev**
image

### Motivation and context

With IAM integration users don't see the tasks with jobs assigned to them anymore, so, dedicated page with a list of job is necessary for annotators. 

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@ActiveChooN 

Could you please review this PR ASAP? It is important to update cvat.org

File: cvat-core/src/server-proxy.js [hunk 1]
```diff
@@ -924,14 +924,24 @@
                 return createdTask[0];
             }
 
-            async function getJob(jobID) {
+            async function getJobs(jobID = null, filter = {}) {
```
Comment (ActiveChooN) [inline cvat-core/src/server-proxy.js:927]: `getTasks` and `getProjects` accept only one argument. Maybe this function can be made the same way?

**Review by ActiveChooN — COMMENTED** [review]
изображение

Equal paddings could look better, I guess.

File: cvat-ui/src/base.scss [hunk 2]
```diff
@@ -22,6 +22,7 @@ $border-color-1: #c3c3c3;
 $border-color-2: rgb(240, 240, 240);
 $border-color-3: #242424;
 $border-color-hover: #40a9ff;
+$card-id-background-color: #1890ff;
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-ui/src/base.scss:25]: Yes, outdated code

File: cvat-core/src/server-proxy.js (hunk 1 above)
Comment (bsekachev) [reply cvat-core/src/server-proxy.js:927]: Updated, thx

**Comment by nmanovic** [comment]
@bsekachev , let's merge and improve the simplified version later.

**Review by nmanovic — APPROVED** [review]

**Pull request merged**